apps/condo/domains/notification/constants/constants.js

```diff
@@ -196,6 +198,16 @@ const MESSAGE_META = {
             residentId: { defaultValue: '', required: true },
         },
     },
+    [BILLING_RECEIPT_AVAILABLE_MANUAL_TYPE]: {
+        dv: { defaultValue: '', required: true },
```

Comment:
Why defaultValue is '' ? I think it's wrong to use it like that...since it falsey and similar to null....and if it's similar to null, why it's required?